api/server/router/network/filter.go

```diff
@@ -73,6 +94,28 @@ func filterNetworks(nws []types.NetworkResource, filter filters.Args) ([]types.N
 		displayNet = append(displayNet, nw)
 	}
 
+	var danglingOnly bool
+	if filter.Contains("dangling") {
+		fErr := filter.WalkValues("dangling", func(fval string) error {
+			switch fval {
+			case "true", "1":
+				danglingOnly = true
+			case "0", "false":
```

Comment:
Wonder if we should be explicit here, and add `danglingOnly = false`. Perhaps because I'm used to "fall-through" being the default in other languages, but I had to look twice to interpret this correctly 😊

Reply:
It's already false?

Reply:
Yeah, I know, it's probably just me, reading;

```
switch {
  case foo:
    dofoo()
  case bar:
    dobar()
  case baz:
  default:
    dosomethingelse()
}
```

I always tend to first read that as `baz:` falls through to `default:` (so `error` in this case)

Reply:
Actually; should this use `filter.ExactMatch()` here? https://github.com/moby/moby/blob/2a7388a6c47a58c402acfe0059fbe27a7a090c23/daemon/prune.go#L185-L191

Reply:
I'm not sure why one would be better than the other. This filters package is pretty confusing.